app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntryLogicTest.kt

```diff
@@ -889,6 +890,133 @@ class BloodPressureEntrySheetLogicTest {
     verifyNoMoreInteractions(ui)
   }
 
+  @Test
+  fun `when a different user clicks on save while updating a BP, then the updated BP measurement should be saved with the user's ID and the corresponding facility ID`() {
+    // given
+    reset(userSession, facilityRepository)
+    val userFromDifferentFacility = PatientMocker.loggedInUser(uuid = UUID.fromString("4844b826-a162-49fe-b92c-962da172e86c"))
+    val differentFacility = PatientMocker.facility(uuid = UUID.fromString("f895b54f-ee32-4471-bc0c-a91b80368778"))
+
+    whenever(userSession.requireLoggedInUser()).doReturn(Observable.just(userFromDifferentFacility))
+    whenever(facilityRepository.currentFacility(userFromDifferentFacility)).doReturn(Observable.just(differentFacility))
+
+    val oldCreatedAt = LocalDate.of(1990, 1, 13).toUtcInstant(testUserClock)
```

Comment:
We have a live template, `instant`, in the current tooling plugin which will generate something like `Instant.parse("2018-01-01T00:00:00Z")`.

We can use this instead of going this round about way.

Reply:
I can make this change and push if you like.

Reply:
Did you do this intentionally to make the behaviour closer to production (converting an entered `LocalDate` to an `Instant`)?

Reply:
I picked up examples from the earlier tests to do this. Hence, this pattern.

Reply:
Yeah, the older tests were written before the tooling plugin. Not a problem though, we can always start using the live templates for newer tests.

Reply:
Fixed.